### PR TITLE
Start including MASP randomness in the test vectors.

### PR DESCRIPTION
**Pull request by murisi**
## Describe your changes
Augment the MASP test vectors with the used randomness parameters. Also add the schema for the randomness parameters to the `tx-schema` example.

## Indicate on which release or other PRs this topic is based on
Namada v0.26.1
https://github.com/anoma/masp/pull/84

## Checklist before merging to `draft`
- [x] I have added a changelog
- [x] Git history is in acceptable state

### CODE REVIEW

**Review by grarco — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -124,9 +124,9 @@ ledger-transport-hid = "0.10.0"
 libc = "0.2.97"
 libloading = "0.7.2"
 linkme = "0.3.24"
-# branch = "main"
-masp_primitives = { git = "https://github.com/anoma/masp", rev = "3aacc707c5948e7423589ac617305448bead9842" }
-masp_proofs = { git = "https://github.com/anoma/masp", rev = "3aacc707c5948e7423589ac617305448bead9842", default-features = false, features = ["local-prover"] }
+# branch = "murisi/receive-randomness-with-borsh2"
```
Comment (cwgoes) [inline Cargo.toml:127]: Can we merge this to main?

File: Cargo.toml [hunk 2]
```diff
@@ -124,9 +124,9 @@ ledger-transport-hid = "0.10.0"
...
+masp_primitives = { git = "https://github.com/anoma/masp", rev = "3c9a61895ebf21ed9f60ce40c481ee932c719b71" }
+masp_proofs = { git = "https://github.com/anoma/masp", rev = "3c9a61895ebf21ed9f60ce40c481ee932c719b71", default-features = false, features = ["local-prover"] }
```
Comment (cwgoes) [inline Cargo.toml:129]: Is this merged to main?

**Review by cwgoes — DISMISSED** [review]
ACK, but we should merge the upstream first

File: Cargo.toml (hunk 1 above)
Comment (murisi) [reply Cargo.toml:127]: Yes, I've just merged this to `main`. Now updating this branch...

File: Cargo.toml (hunk 2 above)
Comment (murisi) [reply Cargo.toml]: This has now been merged to `main` and the commit hash has been updated accordingly.

**Pull request merged**